app/controllers/bookmarks/search_controller.rb

```diff
@@ -2,18 +2,38 @@ class Bookmarks::SearchController < ApplicationController
   # GET /bookmarks/search
   # GET /bookmarks/search.json
   def index
-    @bookmarks = BookmarksIndex.query(
-      bool: {
-        should: [
-          { match: { title: params[:q] } },
-          { match: { tags: params[:q] } }
-        ]
-      }
-    ).objects.page params[:page]
+    @query = params[:q]
+    @searcher = policy_scope(BookmarkSearcher)
+
+    if @query.blank? || @query.empty?
+      @bookmarks = Bookmark.none
+    else
+      if @query
+        odd_quotes = @query.count('"').odd?
```

Comment:
no love for the single quote?

Reply:
Nope, figured it was going to be not too fun with apostrophes that I'd rather just not deal with them for now until I see how well this works